### DISCUSSION TITLE
webdav sync with mailbox.org

### DISCUSSION THREAD

**Discussion by micalx** [super-productivity/super-productivity#2814] [Q&A] [answered]
Hello everyone,

I try to sync SuperProductivity (SP) with my account at mailbox.org which is based on OpenXchange.

I tried several different settings especially with the URL and calendar name but I don't get it to work. It is always the same error message: "CalDAV: Request failed because of a client side network error"

Can anyone help?

Thank you & kind regards
Michael

**Comment by Lichtwitzig** [comment]
Hi!
use the following URL:
https://dav.mailbox.org/servlet/webdav.infostore/Userstore/FIRSTNAME%20LASTNAME
Greetings!

**Reply by micalx** [reply]
Hi @Lichtwitzig ,
*thank you for your message!*

When I use this URL, I get an "website can't be reached" aka http 404 (?)

I struggle a little bit with the "%20" in the URL. When I use a "." or nothing

- https://dav.mailbox.org/servlet/webdav.infostore/Userstore/FIRSTNAME.LASTNAME
- https://dav.mailbox.org/servlet/webdav.infostore/Userstore/FIRSTNAMELASTNAME

=> There is a message "There is nothing here, sorry."

Do you use the CalDav feature with mailbox.org?

Regards
Michael

**Reply by Lichtwitzig** [reply]
Just leave the %20 as is.
Yes, I do use CalDAV + SP, though I want to change away from Mailbox in the next years

**Reply by micalx** [reply]
Hello again,

I still can't make the CalDav sync work :/ I tried a lot of combinations and there is also a help for "connect your device" within mailbox.org 

removed images

Last thing I forgot to mention, I have 2FA enabled and therefore I tried an app specific password. Same here :(

Still no success

Any further help?

Thank you in advance & kind regards
Michael

**Comment by Lichtwitzig** [comment]
Hi, 
I think with 2FA it won't work, because SP lacks the possibility to login with both factors.
So your URL would be:
https://dav.mailbox.org/servlet/webdav.infostore/Userstore/michael%20alexander
(if Alexander ist your lastname)
Though I have to say, that WebDAV got faulty with a new version in my case too,  thus I'm using file sync now, using a Pi with Syncthing.
Greetings, LW

**Comment by jpgaubier** [comment]
Add a folder in your Mailbox.org matching the name of whatever you have entered in "Sync Folder Path" under the Sync settings. The sync cannot create the folder, so the sync fails if it can't find it.

**Reply by iamuserone** [reply]
Please provide an example url.

**Comment by flippinhutt** [comment]
Did this ever did figured out or fixed?

**Comment by cdanne** [comment, marked as answer]
I've set it up with my mailbox account and it worked out-of-the box.

- **URL** including user name like https://dav.mailbox.org/servlet/webdav.infostore/Userstore/Firstname%20Lastname/
- Primary mailbox.org email address as **username**
- If you use 2FA create an **application password** in mailbox.org first and use that instead of regular mailbox.org password https://kb.mailbox.org/de/privat/sicherheit-und-privatsphaere/applikationspasswoerter-fuer-externe-programme/
- ensure sync folder path (default super-productivity) exists

See https://kb.mailbox.org/de/privat/drive/webdav-unter-windows/

**Reply by micalx** [reply]
yes, it's working for me as well
